CHANGELOG.rst

```diff
@@ -1,3 +1,9 @@
+1.14.2 (2023-04-03)
```

Comment:
Given we are adding new units (as opposed to fixing bugs), we should increment the minor version:

```suggestion
1.15.0 (2023-04-03)
```